File: hyperarchy/app/models/election_visit.rb
class ElectionVisit < Prequel::Record
  column :id, :integer
  column :election_id, :integer
  column :user_id, :integer
  column :created_at, :datetime
  column :updated_at, :datetime

  belongs_to :election
  belongs_to :user

  def organization_ids
    election ? election.organization_ids : []
  end
end

File: client_src/javascripts/app/models/election_visit.js
_.constructor("ElectionVisit", Model.Record, {
  constructorInitialize: function() {
    this.columns({
      userId: 'key',
      electionId: 'key',
      updatedAt: 'datetime'
    });

    this.belongsTo('election');
    this.belongsTo('user');
  }
});
